qiskit/dagcircuit/_dagcircuit.py

```diff
@@ -1140,7 +1150,7 @@ def remove_nondescendants_of(self, node):
                 self._remove_op_node(n)
 
     def layers(self):
-        """Yield a layer for all d layers of this circuit.
+        """Return a list of layers for all d layers of this circuit.
```

Comment:
Can you please check if this was intentional as well and that everything is OK in the git history of the branch? It seems to be reverting the changes made by #371

Reply:
sorry this was a mistake. restored.